torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -29,23 +30,25 @@ namespace {
 
 std::string stringSlice(
```

Comment:
Actually, sorry, one more comment: just as we updated the list implementation to work with negative step we should do the same with strings right ?